rakelib/utils.rake

```diff
@@ -68,6 +68,7 @@ class Utils
 
   # select the xcode version we want/support
   def self.version_select
+    Rake.sh 'mdimport /Applications/Xcode* && sleep 1'
```

Comment:
What @AliSoftware means is that an arbitrary sleep might not fix it on all systems, as we never know how long indexing will take. 

I think it might be better to write our own fallback search in case spotlight doesn't do it for us. It doesn't have to be extensive, we can always fail and tell the user the reason, and it's up to them to then fix spotlight on their device.

Another thing is that with this line, the `version_select` call will always at least take 1s to execute, slowing down the whole build process. Might be best to make this somehow static (like a swift static constant, my ruby-fu is failing me right now 😆 ) to cache the result.